docs/index.rst

```diff
@@ -3,53 +3,66 @@
 .. only:: html
 
    .. note::
-       Parsec's documentation is available in several languages and versions.
-       Expand the "Read the Docs" panel at the bottom of the sidebar to see the list.
+        Parsec documentation is available in English and French.
+        Expand the "Read the Docs" panel at the bottom of the sidebar to see the list.
 
-
-===========
-Parsec Docs
-===========
+====================
+Parsec Documentation
+====================
 
 .. image:: parsec_doc_logo.png
     :align: center
     :alt: parsec banner
+    :class: extra-margin-bottom·
 
-Welcome to the Parsec documentation. If you are new to this documentation,
-we recommend that you read the :ref:`introduction page <doc_introduction>` to get
-an overview of what this documentation has to offer.
+Welcome to the Parsec documentation. Take a look at the :ref:`Introduction <doc_introduction>`
+to get an overview of what Parsec can do for you.
 
 The table of contents displayed below and in the sidebar should let you easily
-access the documentation for your topic of interest. You can also use the search
-function in the top left corner.
-
-The main documentation for the site is organized into the following sections:
+navigate the documentation.
+You can also use the search function in the top left corner.
 
 .. toctree::
     :maxdepth: 1
-    :caption: General
-    :name: sec-general
+    :caption: User Guide
+    :name: section-user-guide
 
     introduction
-    userguide/index
-    adminguide/index
+    userguide/installation
+    userguide/new_organization
+    userguide/manage_organization
+    userguide/join_organization
+    userguide/first_steps
```

Comment:
```suggestion
    userguide/new_organization
    userguide/first_steps
    userguide/manage_organization
    userguide/join_organization
```

I would move `first_step` just after `new_orga` since it's referenced multiple time on this page.

Reply:
[`8af7270`](https://github.com/Scille/parsec-cloud/pull/8387/commits/8af72704163b00dc0eb1006ad9ed89e521f1731f)